FrameworkSystem/DB/InstalledComponentsDB.py

```diff
@@ -239,6 +240,95 @@ def toDict( self, includeComponents = False, includeHosts = False ):
+  # status
+  DIRAC = Column( 'DIRACVersion', String( 32 ) )
+  Load1 = Column( 'Load1', String( 32 ) ) # float
+  Load5 = Column( 'Load5', String( 32 ) ) # float
+  Load15 = Column( 'Load15', String( 32 ) ) # float
+  Memory = Column( 'Memory', String( 32 ) )
+  DiskOccupancy = Column( 'DiskOccupancy', String( 32 ) )
+  Swap = Column( 'Swap', String( 32 ) )
+  CPUClock = Column( 'CPUClock', String( 32 ) ) # float
+  CPUModel = Column( 'CPUModel', String( 64 ) )
+  CertificateDN = Column( 'CertificateDN', String( 64 ) )
+  CertificateIssuer = Column( 'CertificateIssuer', String( 64 ) )
+  CertificateValidity = Column( 'CertificateValidity', String( 64 ) )
+  Cores = Column( 'Cores', Integer )
+  PhysicalCores = Column( 'PhysicalCores', Integer )
+  OpenFiles = Column( 'OpenFiles', Integer )
+  OpenPipes = Column( 'OpenPipes', Integer )
+  OpenSockets = Column( 'OpenSockets', Integer )
+  Setup = Column( 'Setup', String( 32 ) )
+  Uptime = Column( 'Uptime', String( 64 ) )
+  Timestamp = Column( 'Timestamp', DateTime )
+
+  def __init__( self, host = null(), **kwargs ):
+    self.hostName = host
+    fields = dir( self )
+
+    for key, value in kwargs.iteritems():
+      if key in fields and not re.match( '_.*', key ):
+        setattr( self, key, value )
+
+  def fromDict( self, dictionary ):
+    """
+    Fill the fields of the HostLogging object from a dictionary
+    """
+    fields = dir( self )
+
+    try:
+      for key, value in dictionary.iteritems():
+        if key in fields and not re.match( '_.*', key ):
+          setattr( self, key, value )
+    except Exception, e:
+      return S_ERROR( e )
+
+    return S_OK( 'Successfully read from dictionary' )
+
+  def toDict( self ):
+    """
+    Return the object as a dictionary
+    """
+
+    dictionary = {
+                  'HostName': self.hostName,
+                  'DIRAC': self.DIRAC,
+                  'Load1': self.Load1,
+                  'Load5': self.Load5,
+                  'Load15': self.Load15,
+                  'Memory': self.Memory,
+                  'DiskOccupancy': self.DiskOccupancy,
+                  'Swap': self.Swap,
+                  'CPUClock': self.CPUClock,
+                  'CPUModel': self.CPUModel,
+                  'CertificateDN': self.CertificateDN,
+                  'CertificateIssuer': self.CertificateIssuer,
+                  'CertificateValidity': self.CertificateValidity,
+                  'Cores': self.Cores,
+                  'PhysicalCores': self.PhysicalCores,
+                  'OpenFiles': self.OpenFiles,
+                  'OpenPipes': self.OpenPipes,
+                  'OpenSockets': self.OpenSockets,
+                  'Setup': self.Setup,
+                  'Uptime': self.Uptime,
+                  'Timestamp': self.Timestamp
+                  }
+
+    return S_OK( dictionary )
+
 class InstalledComponentsDB( object ):
   """
   Class used to work with the InstalledComponentsDB database.
```

Comment:
Well, this was a joke. "Class used to work" in english can mean "class was working in the past"